ME-MYSELF-I/Weekly-meetings.md

```diff
@@ -5,6 +5,46 @@
 * [29th September 2017](#date-29th-september-2017)
 * [8th October 2017](#date-8th-october-2017)
 * [15th October 2017](#date-15th-october-2017)
+* [22nd October 2017](#date-22nd-october-2017)
+
+### Date: 15th October 2017
+
+#### Who did you work directly with (i.e. we helped each other) this week?
+
+#### Who did you help this week?
+
+* some nipype users: neurostar
+
+#### Who helped you this week?
+
+#### What did you achieve?
+
+* fixing a few more tests in traitlets PR
+* changing TraitedSpec, so it doesn't change mutable variables used as inputs
+* helping with cmtk.nbs:
+  - nipype PR improving some error massages and adding tests for cmtk.nbs
+  - PR to connectomviewer
+
+
+#### What did you struggle with?
+
+* having problem with testing connectomviewer with docker container based on neurodebian (at the end wasn't important for the code I wanted to test to solve an issue)
+
+#### What else did you do this week week?
+
+
+#### What would you like to work on next week?
+
+* continue working on traitlets
```

Comment:
can we set a timeline for traitlets?

Reply:
would you find some time do discus the problem with `DynamicTraitedSpec`. It's not clear to me how important it is to keep it and what would be the best way to do it.

Reply:
`dynamictraitedspec` is important since many interfaces in io and utils depend on it